app/src/main/java/org/thoughtcrime/securesms/home/search/GlobalSearchViewModel.kt

```diff
@@ -41,14 +48,19 @@ class GlobalSearchViewModel @Inject constructor(private val searchRepository: Se
         _queryText
                 .buffer(onBufferOverflow = BufferOverflow.DROP_OLDEST)
                 .mapLatest { query ->
+                    // Minimum search term is 2 characters
```

Comment:
iOS lets us search 1 character.

It's not necessarily a good search, but you can give yourself a 1-char name, so you should be able to search that... and ideally a match of an entire word should be elevated.

Reply:
Yeah, fair enough - have allowed 1 char searches and it does prioritise contacts & groups over individual messages.